test/e2e/quick_start.go

```diff
@@ -70,6 +71,19 @@ type QuickStartSpecInput struct {
 	// which unblocks CNI installation, and for the control plane machines to be ready (after CNI installation).
 	ControlPlaneWaiters clusterctl.ControlPlaneWaiters
 
+	// ExtensionConfigName is the name of the ExtensionConfig. Defaults to "quick-start".
```

Comment:
The additions in this file are mostly copied from scale.go and are only there to deploy the ExtensionConfig if RuntimeSDK is used